### PR TITLE
Need a code review

### PR DESCRIPTION
**Pull request by CHelfgott**
https://github.com/ArnholdInstitute/DHS-Data-Sets/compare/craig-dev?expand=1#diff-28f4b424d2f0d2bba1172410d1427a0d

### CODE REVIEW

File: add_full_index_hash.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  query = "SELECT column_name FROM information_schema.columns "
+  query += "WHERE table_name = \"" +table_name+ "\" AND (column_name ILIKE \"%"
+  query += "%\" OR column_name ILIKE \"%".join(INDEX_COLUMNS)
+  query += "%\")"
+  return query
+  
+
+def main():
+  pg_username = input("Please enter Postgres username:")
+  pg_password = input("Password:")
```
Comment (klandell) [inline add_full_index_hash.py]: Please use a function that will mask the password.

File: add_full_index_hash.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    
+  pg_login = pg_username + ":" + pg_password
+  pg_conn_str = 'postgresql://' + pg_login + '@localhost:5432/dhs_data'
```
Comment (klandell) [inline add_full_index_hash.py]: Should localhost be hardcoded?

**Review by klandell — CHANGES_REQUESTED** [review]
Seems okay, I'm pretty rusty on my python so I can't comment too much on the actual syntax.  The logic seems to make sense though.  

In general there is a lot of commented code.  Can it be removed?  The history will still be there since it is git.

File: add_full_index_hash.py (hunk 1 above)
Comment (CHelfgott) [reply add_full_index_hash.py]: This is intended to be run locally on the AIGH machine.

File: add_full_index_hash.py (hunk 2 above)
Comment (CHelfgott) [reply add_full_index_hash.py]: This is intended to be run locally on the AIGH machine.  (And only once at that.)

**Review by klandell — APPROVED** [review]

**Pull request merged**